changelogs/fragments/122-api.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - api_modify, api_info - support new fields and paths available in ROS7 under ``system ntp``
```

How about explicitly mentioning the new ones:
```suggestion
  - api_modify, api_info - support paths ``system ntp client servers`` and ``system ntp server`` available in ROS7,
    as well as new fields ``servers``, ``mode``, and ``vf`` for ``system ntp client``
```